pkg/tools/prefixpool/prefixpool_reader.go

```diff
@@ -44,7 +44,7 @@ func (ph *Reader) init(prefixes []string) {
 
 // NewPrefixPoolReader gets list of excluded prefixes from config file. Starts config file monitoring.
 // Returns pointer to a struct that contains all information
-func NewPrefixPoolReader(path string) *Reader {
```

Comment:
Why we need this change?

Reply:
In monorepo this function returns Prefixpool
https://github.com/networkservicemesh/networkservicemesh/blob/99fc16ce63422783a52c314be3f38cb665b61ed1/sdk/prefix_pool/prefixpool_reader.go#L43
Now we use it only in ExcludedPrefixesService and we don't need Reader

Reply:
ok, let's change it in separate PR. 
In this PR we can use the `reader. Prefixpool` field.

Reply:
ok, done